test/E2ETests/Smoke.E2E.Tests/DataRowTests.cs

```diff
@@ -21,12 +21,31 @@ public void ExecuteOnlyDerivedClassDataRowsWhenBothBaseAndDerviedClassHasDataRow
                 "DataRowTestMethod (BaseString2)",
                 "DataRowTestMethod (BaseString3)",
                 "DataRowTestMethod (DerivedString1)",
-                "DataRowTestMethod (DerivedString2)");
+                "DataRowTestMethod (DerivedString2)",
```

Comment:
This check has become quite long. It would be more readable if we have seperate E2E tests for `DataRowTestMethodWithSomeOptionalParameters` and `DataRowTestMethodWithAllOptionalParameters`